repos/fdbt-site/cypress_tests/cypress/support/helpers.ts

```diff
@@ -528,3 +528,13 @@ export const addFlatFareProductIfNotPresent = (): void => {
         }
     });
 };
+
+export const checkAndRepickService = (): void => {
+    cy.get('body').then(($body) => {
+        if ($body.text().includes('Tcannot create a return for this service')) {
```

Comment:
Is `Tcannot` a typo? 

`cannot create a return for this service` does not match the error detail message, or is it trying to match to some other hidden data in the DOM?

The error detail on the page is `As your service only operates in a single direction, you cannot create a return product for this service` we should either do a full match on that to be explicit, or perhaps just the second half `you cannot create a return product for this service` to allow for future other reasons why a return may not be valid for this service.